controller/api/apis.go

```diff
@@ -3625,3 +3632,30 @@ type REST_SigstoreRootOfTrustCollection struct {
 type REST_SigstoreVerifierCollection struct {
 	Verifiers []REST_SigstoreVerifier `json:"verifiers"`
 }
+
```

Comment:
The following 3 types are not found in controller/api/apis.yaml

Reply:
I'll update the `apis.yaml` file when PR conversations have settled. The proxy support can be implemented.